app/src/main/java/com/example/giftly/handler/FireBaseClient.java

```diff
@@ -160,10 +156,14 @@ public String call() throws Exception {
                 ArrayList<String> participants = (ArrayList<String>) event.get("participants");
 
                 //Check if event already has the user, if not add them and update the doc
-                if (participants == null) participants = new ArrayList<String>(1);
+                if (participants == null)
+                    participants = new ArrayList<String>(1);
                 Log.d(TAG, "Checking Event");
-                if (!participants.contains(getAuth().getUid())) participants.add(getAuth().getUid());
+                if (!participants.contains(getAuth().getUid())) {
+                    participants.add(getAuth().getUid());
                     targetEvent.update("participants", participants);
+                }
+
```

Comment:
lines 159 to 165, maybe use an if-else statement, or there can be a cleaner version to this?